src/main/java/uk/gov/hmcts/ccd/data/roleassignment/RoleAssignmentResponse.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.ccd.data.roleassignment;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+import java.util.List;
+
+@Builder
+@Data
+public class RoleAssignmentResponse {
+    @JsonProperty("roleAssignmentResponse")
+    private List<RoleAssignmentRecord> roleAssignmentRecords;
```

Comment:
roleAssignmentResponse is singular and roleAssignmentRecords is plural, is this ok like this

Reply:
It is the naming the am team is using. roleAssignmentResponse will be a node in json and it will contain a list of records. I know, it might be confusing.